I2C.bme280.oled/simple_remote.py

```diff
@@ -0,0 +1,159 @@
+from luma.core.render import canvas
+from luma.oled.device import ssd1306
+from PIL import ImageFont
+import argparse
+import json
+import logging.handlers
+import smbus2
+import bme280
+import os
+import time
+from dateutil import tz
+import requests
+from datetime import datetime, timezone, timedelta
+
+parser = argparse.ArgumentParser()
+
+# Rev 2 Pi, Pi 2 & Pi 3 uses bus 1
+# Rev 1 Pi uses bus 0
+# Orange Pi Zero uses bus 0 for pins 1-5 (other pins for bus 1 & 2)
+parser.add_argument('--bus', '-b', default=0, type=int, help='Bus Number, defaults to 0')
+
+# IP address of your YANPIWS device you want to show data from
+parser.add_argument('--remote_ip', '-ip', default='192.168.68.105', type=str, help='Temp sensor ID, defaults to 0x76')
+
+# ID from your YANPIWS config.csv of temp 1
+parser.add_argument('--temp_id1', '-id1', default='143', type=int, help='remote temp ID #1, defaults to 143')
+
+args = parser.parse_args()
+
+# Build URL and set some vars
+yanpiws_ip = args.remote_ip
+yanpiws_temp_1 = args.temp_id1
+yanpiws_ajax_url = 'http://' + str(yanpiws_ip) + '/ajax.php?content='
+last_seen_temp= 0.0
+
+# set up syslog logging
+my_logger = logging.getLogger('MyLogger')
+my_logger.setLevel(logging.DEBUG)
+handler = logging.handlers.SysLogHandler(address='/dev/log')
+my_logger.addHandler(handler)
+
+# BME280 sensor address (default address), Initialize I2C bus and calibration
+address = 0x76 # can also be 0x77 - check  i2cdetect -y 1 or i2cdetect -y 0
+bus = smbus2.SMBus(1) # can also be 0, depends on where you found device on per i2cdetect
+calibration_params = bme280.load_calibration_params(bus, address)
+
+def post_to_yanpiws():
+    data = bme280.sample(bus, address, calibration_params)
+    temperature_celsius = data.temperature
+    temperature_fahrenheit = round((temperature_celsius * 9 / 5) + 32, 2)
+    url = 'http://' + str(yanpiws_ip) + '/parse_and_save.php'
+    data = {
+        "model" : "BMP280",
+        "time" : datetime.now(tz=tz.tzlocal()).strftime("%Y-%m-%d %H:%M:%S"),
+        "id" : 71, # E bedroom, todo - not hardcode
+        "temperature_F" : temperature_fahrenheit,
+        "password" : "boxcar-spinning-problem-rockslide-scored"  # default pass, todo - not hardcode
+    }
+    global last_seen_temp
+    last_seen_temp = temperature_fahrenheit
+    requests.post(url, data=data)
+
+def get_string_from_url(url):
+    import urllib.request
+    raw_html = urllib.request.urlopen(url).read().decode('utf-8').rstrip()
+    return raw_html
+
+
+def get_remote_humid_and_temp(id):
+    forecast_url = yanpiws_ajax_url + 'humidity&id=' + str(id)
+    return json.loads(get_string_from_url(forecast_url))
+
+
+def get_remote_forecast():
+    forecast_url = yanpiws_ajax_url + 'forecast_full_json'
+    return json.loads(get_string_from_url(forecast_url))
+
+
+def get_remote_sun():
+    data = json.loads(get_string_from_url(yanpiws_ajax_url + "sunrise"))
+    data.update(json.loads(get_string_from_url(yanpiws_ajax_url + "sunset")))
```

Comment:
`requests` module has a [built-in json parser](https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content). Using `requests` to pull the data and parse should allow you to drop the `import json` at the top.